### ISSUE TITLE
`invariant_n!` macro does not support an invocation with only 4 arguments.

### ISSUE THREAD

**Issue by codyjrivera** [anvil-verifier/anvil#536]
When I tried to invoke `invariant_n!` with only 4 arguments, the compiler gives me an `unexpected end of macro invocation` error. The comment around the macro implies that you can supply only four arguments if you wanted to.

This is an example of offending input (a workaround is to put `true_pred()` as the fifth argument):
```rust
invariant_n!(
      spec, lift_action(stronger_next), 
      lift_action(VRSCluster::every_new_req_msg_if_in_flight_then_satisfies(requirements)),
      lift_action(VRSCluster::next())
  );
```

**Comment by marshtompsxd** [comment]
I will look into this issue.

**Comment by marshtompsxd** [comment]
Addressed by #548
